src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/VedtaksstatistikkDvh.kt

```diff
@@ -0,0 +1,205 @@
+package no.nav.tilleggsstonader.sak.statistikk.vedtak
+
+import no.nav.tilleggsstonader.sak.opplysninger.pdl.dto.AdressebeskyttelseGradering
+import no.nav.tilleggsstonader.sak.statistikk.vedtak.StønadstypeDvh.BARNETILSYN
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vilkårsresultat
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.RegelId
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.AktivitetType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.MålgruppeType
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.ResultatVilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.Vilkårperiode
+import no.nav.tilleggsstonader.sak.vilkår.vilkårperiode.domain.VilkårperiodeType
+import java.time.LocalDate
+import java.time.ZonedDateTime
+import java.util.UUID
+
+// TODO: Vurder om dette bør flyttes til kontrakter
+
+class VedtaksstatistikkDvh(
+    val fagsakId: UUID,
+    val behandlingId: UUID,
+    val eksternBehandlingId: Long,
+    val relatertBehandlingId: Long?, // Ekstern behandlingsid på relatert behandling
+    val adressebeskyttelse: AdressebeskyttelseDvh,
+    val tidspunktVedtak: ZonedDateTime,
+    val målgruppe: List<MålgruppeDvh>,
+    val aktivitet: List<AktivitetDvh>,
+    val vilkårsvureringer: List<VilkårsvurderingDvh>,
+    val person: String,
+    val barn: List<BarnDvh>,
+    val behandlingType: BehandlingTypeDvh,
+    val behandlingÅrsak: BehandlingÅrsakDvh,
+    val vedtakResultat: VedtakResultatDvh,
+    val vedtaksperioder: VedtaksperioderDvh,
+    val utbetalinger: List<UtbetalingDvh>,
+    val stønadstype: StønadstypeDvh = BARNETILSYN,
+    val kravMottatt: LocalDate,
+    val årsakRevurdering: ÅrsakRevurderingDvh? = null,
+    val avslagÅrsak: AvslagÅrsakDvh? = null,
+
+)
+
+enum class StønadstypeDvh {
+    BARNETILSYN,
+}
+
+data class UtbetalingDvh(
+    val gyldigFom: LocalDate,
+    val gyldigTom: LocalDate,
+    val beløp: Int,
+    val satstype: SatstypeDvh = SatstypeDvh.DAGLIG,
+)
+
+enum class SatstypeDvh {
+    DAGLIG,
+}
+
+data class VedtaksperioderDvh(
+    val fomDato: LocalDate,
+    val tomDato: LocalDate,
+    val utgifter: Int,
+)
+
+enum class VedtakResultatDvh {
+    INNVILGET,
+    AVSLÅTT,
+    OPPHØRT,
+}
+
+enum class BehandlingÅrsakDvh {
+    KLAGE,
+    NYE_OPPLYSNINGER,
+    SØKNAD,
+    MIGRERING,
+    KORRIGERING_UTEN_BREV,
+    PAPIRSØKNAD,
+    SATSENDRING,
+    MANUELT_OPPRETTET,
+}
```

Comment:
Tror vi kan sløyfe noen av disse
```suggestion
enum class BehandlingÅrsakDvh {
    KLAGE,
    NYE_OPPLYSNINGER,
    SØKNAD,
    PAPIRSØKNAD,
    MANUELT_OPPRETTET,
}
```

Reply:
Hvordan tenker du de fjerna årsakene skal mappes til, da? 
Altså siste linja her: 
```
    companion object {
        fun fraDomene(årsak: BehandlingÅrsak) = when(årsak) {
            BehandlingÅrsak.KLAGE -> KLAGE
            BehandlingÅrsak.NYE_OPPLYSNINGER -> NYE_OPPLYSNINGER
            BehandlingÅrsak.SØKNAD -> SØKNAD
            BehandlingÅrsak.PAPIRSØKNAD -> PAPIRSØKNAD
            BehandlingÅrsak.MANUELT_OPPRETTET -> MANUELT_OPPRETTET
            BehandlingÅrsak.MIGRERING, BehandlingÅrsak.KORRIGERING_UTEN_BREV, BehandlingÅrsak.SATSENDRING -> TODO()
        }
    }
```